### ISSUE TITLE
SMB 2/3

### ISSUE THREAD

**Issue by j-m-baker** [AgNO3/jcifs-ng#124]
Hello

Does this version of jcifs support SMB 2/3?

**Comment by mbechler** [comment]
Yes, see https://github.com/AgNO3/jcifs-ng/blob/master/README.md.
It does not support all of the protocol features, but should be usable with servers only supporting the higher protocol versions (as long as encryption is not requried).
